src/go/rpk/pkg/cli/cluster/quotas/quotas.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package quotas
+
+import (
+	"strings"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/types"
+)
+
+func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "quotas",
+		Aliases: []string{"quota"},
+		Args:    cobra.NoArgs,
+		Short:   "Manage Redpanda client quotas",
```

Comment:
question. Parent commands don't have long help texts?

Reply:
Not necessarily, it depends on the complexity of the command and whether we want to explain a concept there.